lightning/pkg/server/lightning.go

```diff
@@ -27,6 +27,7 @@ import (
 	"encoding/pem"
 	"fmt"
 	"io"
+	mathrand "math/rand"
```

Comment:
I suggest we use "crypto/ rand". Because the issue wants to avoid that if two lightning have same `time.Now().UnixNano()` they will have the same ID. Now if math/rand is initialized by the same `time.Now().UnixNano()` they still have the same ID.